lib/vector-vrl-functions/src/remove_metadata_field.rs

```diff
@@ -44,11 +44,21 @@ impl Function for RemoveMetadataField {
 
     fn compile(
         &self,
-        _state: (&mut state::LocalEnv, &mut state::ExternalEnv),
+        state: (&mut state::LocalEnv, &mut state::ExternalEnv),
```

Comment:
Is it possible to destructure here like

```suggestion
        (_, external_env): (&mut state::LocalEnv, &mut state::ExternalEnv),
```

so we don't have to read as `state.1` below?